### PR TITLE
Improved the README file and pip_requirements

### PR DESCRIPTION
**Pull request by iugrina**
Dear *,

I've improved the README file and adjusted pip_requirements file to better suit reproducible research.

Best,
Ivo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,143 @@ American-Gut
...
+## Basics
+
+Although American-Gut repo provides separate scripts (`scripts` folder)
+and a package (`americangut` folder) it is primarily intended to be used
+through notebooks (`ipynb` folder).
+
+There are a few environment variable that can be used to customize the run:
+
+- AG\_TESTING: if set to `True` scripts will not download AmericanGut
+  EBI data (ERP012803) bit instead work with test data (subset of the original
```
Comment (wasade) [inline README.md]: bit -> but

File: README.md [hunk 2]
```diff
@@ -2,3 +2,143 @@ American-Gut
...
+Markdown based notebooks can be found in `./ipynb/primary-processing/` folder
+and have extension `md`.
+To use these notebooks we first need to create a profile for `ag_ipymd`
+with
+
+```bash
+$ ipython profile create ag_ipymd
+```
+
+and adjust newly created `/path/to/.ipython/profile_ag_ipymd/ipython_notebook_config.py'
```
Comment (wasade) [inline README.md]: the end of this line uses a single quote (') but should use a back tick (`)

File: README.md [hunk 3]
```diff
@@ -2,3 +2,143 @@ American-Gut
...
+by addding
```
Comment (wasade) [inline README.md]: addding -> adding

File: pip_requirements.txt [hunk 4]
```diff
@@ -1,8 +1,20 @@
-click
+IPython < 4.0.0
```
Comment (wasade) [inline pip_requirements.txt]: @jwdebelius, we currently have IPython < 4.0.0 in our conda requirements, but iirc, you've been operating without issue on IPython > 4.0.0, correct?

File: README.md (hunk 1 above)
Comment (iugrina) [inline README.md]: resolved

File: README.md (hunk 2 above)
Comment (iugrina) [inline README.md]: resolved

File: README.md (hunk 3 above)
Comment (iugrina) [inline README.md]: resolved

**Pull request merged**